helper/rollup/remote.go

```diff
@@ -141,6 +141,36 @@ func remoteLoad(addr string, tlsConf *tls.Config, table string) (*Rules, error)
 		},
 		nil,
 	)
+	if err != nil {
+		query = `SELECT
+			regexp,
+			function,
+			age,
+			precision,
+			is_default
+		FROM system.graphite_retentions
+		ARRAY JOIN Tables AS table
+		WHERE (table.database = '` + db + `') AND (table.table = '` + table + `')
+		ORDER BY
+			is_default ASC,
+			priority ASC,
+			regexp ASC,
+			age ASC
+		FORMAT JSON`
+
+		body, _, _, err = clickhouse.Query(
+			scope.New(context.Background()).WithLogger(zapwriter.Logger("rollup")).WithTable("system.graphite_retentions"),
+			addr,
+			query,
+			clickhouse.Options{
+				Timeout:        10 * time.Second,
```

Comment:
Shouldn't we made this configurable? Or move into constant

Reply:
Query is simple. No need to be configurable, move it to constant